src/index.html

```diff
@@ -1,13 +1,432 @@
+                      alt="Place 1"
+                      class="place__photo"
+                    >
+                  </div>
+
+                </div>
+              </div>
+            </div>
+            <div class="place__description">
+              <a
+                rel = "noopener"
+                href="https://goo.gl/maps/wptmGcTmj6bHzoyv7"
+                target="_blank"
+                class="place__link"
+               >
+                Marina Palms
+              </a>
+              / North Miami Beach, FL 33162
+            </div>
+          </div>
+          <div class="gallery__place place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-6">
+                  <div class="place__photo-container">
+                    <img
+                      src="images/photos/3.jpg"
+                      alt="place 1"
+                      class="place__photo"
+                    >
+                  </div>
+
+                </div>
+                <div class="grid__item grid__item--desktop-7-12">
+                  <div class="place__photo-container">
+                    <img
+                      src="images/photos/4.jpg"
+                      alt="Place 1"
+                      class="place__photo"
+                    >
+                  </div>
+
+                </div>
+              </div>
+            </div>
+            <div class="place__description">
+              <a
+                rel = "noopener"
+                href="https://goo.gl/maps/XzikKpCSYQ2NiYMq7"
+                target="_blank"
+                class="place__link"
+                >
+                Edition Residens
+              </a>
+              / Miami Beach, FL 33139
+            </div>
+          </div>
+          <div class="gallery__place place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-4">
+                  <div class="place__photo-container">
+                    <img
+                      src="images/photos/5.jpg"
+                      alt="place 1"
+                      class="place__photo"
+                    >
+                  </div>
+
+                </div>
+                <div class="grid__item grid__item--desktop-5-12">
+                  <div class="place__photo-container">
+                    <img
+                      src="images/photos/6.jpg"
+                      alt="Place 1"
+                      class="place__photo"
+                    >
+                  </div>
+
+                </div>
+              </div>
+            </div>
+            <div class="place__description">
+              <a
+                rel = "noopener"
+                href="https://goo.gl/maps/AiawzZ1xy2vGtYCe7"
+                target="_blank"
+                class="place__link"
+              >
+                Faena House / Miami Beach, FL 33140
```

Comment:
Remove exact adress from here, you're repeating it after
```suggestion
                Faena House
```